meeting-minutes/client-minutes/march10-client-minutes.md

```diff
@@ -0,0 +1,53 @@
+
+The client asked whether we had a way to remove mentees from our system, and I let the client know that there's nothing directly in our system's UI itself that allows mentees to be removed, however, we can manually remove Mentee users from firebase. The client said that this setup works, and that we won't need to worry about admins being able to remove mentees then.
+
+As briefly noted from Feb 23rd, something that's important to the client/Codebar is security. As such, if time permits, the client would like us to look into the back-end security of our website to make sure that user information isn't able to be queried/accessed. Since we planned our sprint this morning, I let the client know that this may not be something that we're able to deliver before next week. However, it is something that we can keep in mind for the remaining weeks of the course if we have the time. I also mentioned to the client that the course requires us to deliver a final report (where we document our codebase), and so likely our focus will shift to fulfilling remaining course logistics. The client understands that ensuring the security of our back-end might be a large task, and to keep her updated if we're not able to look in this, so that she can let Codebar know. Realistically, this is likely something that we'll include in our final documentation as a limitation if we're unable to ensure the security of our back-end.
+
+Pace B then demonstrated their application. They made progress on the front-end by creating the UI for accepted/declined mentees. At the moment, they're using hard-coded data, but they're hoping to pull information from their database soon. They were also working on the matching algorithm.
+
+Pace A demonstrated their application to show the update to the UI (mainly changing some navigation links to access the various forms, as well as adding the colour scheme to their website). Most of their work has been on the backend, and so they're hoping to connect the back-end with the front-end so that the client will have a visual of what their application can do.
+
+### ACTION ITEMS
+
+3/10-01. Remove the notification upon login stating that the user's suggested mentees have been updated.
+3/10-02. Solidify a course of action regarding the GET request.
+3/10-03. Look into the back-end security of our website.
```

Comment:
Will keep in mind, do our best to make progress on security in the final weeks, and at the very least include the main areas where we think security is lacking and suggest a course of action to fix it, in the final report.